docs/basics.txt

```diff
@@ -1,10 +1,8 @@
 [[basics]]
-JanusGraph Basics
-=================
+= JanusGraph Basics
 
 [[configuration]]
-Configuration
--------------
+== Configuration
 
 A JanusGraph graph database cluster consists of one or multiple JanusGraph instances. To open a JanusGraph instance a configuration has to be provided which specifies how JanusGraph should be set up.
```

Comment:
Reverting this change as this will need further discussion; added an entry to the deferred issue.